assets/armsnippets.jsonc

```diff
@@ -103,9 +103,9 @@
         "prefix": "arm-nested",
         "body": [
             "{",
-            "    \"name\": \"${1:nestedDeployment1}\",",
+            "    \"name\": \"${1:linkedDeployment1}\",",

```

Comment:
Have you looked up the official terminology?  Definitely seeing "nested templates" used for links, e.g. MS docs here: https://docs.microsoft.com/en-us/azure/lab-services/deploy-nested-template-environments.

Reply:
I was trying to set this up for an additional snippet. This one uses a tempalte link. We do not have a snippet that uses an inline tempalte. My thought was that one would be `arm-linked` and the other `arm-nested`. That said, I see here that I only renamed the variable and not resource name and not the snippet :).

I will revert this and then consider naming once we add the new snippet. Perhaps it will end up `arm-nested-linked` and `arm-nested-inline`.

Reply:
Fixed